modules/appnexusBidAdapter.js

```diff
@@ -597,7 +597,7 @@ function newBid(serverBid, rtbBid, bidderRequest) {
     cpm: rtbBid.cpm,
     creativeId: rtbBid.creative_id,
     dealId: rtbBid.deal_id,
-    currency: 'USD',
+    currency: rtbBid.publisher_currency_code || 'USD',
```

Comment:
@astartsky 
If you want the name of the currency (like `USD`) then you should change the field here to `publisher_currency_codename`.

The code field will return the currency symbol (like `$`), which doesn't match the backup value and the unit test.

Reply:
@jsnellbaker thanks for the clarification! I made a fix